README.md

```diff
@@ -207,71 +199,54 @@ local, which is the path on the host and owner and group which are the
 owner and group permissions the directory will be mounted with on the
 guest.
 
-## `controlrepo/.rizzo.json`
-
-```json
-{
-  "defaults": {
-    "bootstrap_script_path": "bootstrap_puppet4.sh",
-    "bootstrap_script_args": "-l -f `hostname -f`",
-    "bootstrap_guest_path": "/tmp/bootstrap",
-    "boot_timeout": 500,
-    "box": "centos7.box",
-    "box_url": "https://vagrantboxes/centos7.box",
-    "box_download_checksum": "3764a2c4ae3829aa4b50971e216c3a03736aafb2",
-    "box_download_checksum_type": "sha1",
-    "memory": "1024",
-    "netmask": "255.255.255.0",
-    "update_packages": true,
-    "update_packages_command": "yum -y update",
-    "shutdown": true,
-    "shutdown_command": "/sbin/shutdown -h now"
-  },
-  "nodes": [
-    {
-      "name": "infra-puppetca",
-      "hostname": "infra-puppetca1.example.org",
-      "forwarded_ports": [
-        {
-          "guest": "8140",
-          "host": "8140"
-        }
-      ],
-      "ip": "172.16.100.5",
-      "memory": "2048"
-    },
-    {
-      "name": "infra-puppet",
-      "hostname": "infra-puppet1.example.org",
-      "ip": "172.16.100.6",
-      "memory": "2048"
-    },
-    {
-      "name": "www-api",
-      "hostname": "www-api1.example.org",
-      "ip": "172.16.100.7"
-    },
-    {
-      "name": "mssql1",
-      "hostname": "mysql1.example.org",
-      "forwarded_ports": [
-        {
-          "guest": "5985",
-          "host": "5985"
-        }
-      ],
-      "ip": "172.16.100.8",
-      "windows": true,
-      "gui": true,
-      "shutdown": false,
-      "update_packages": false,
-      "box": "mwrock/Windows2016",
-      "box_url": "",
-      "box_download_checksum": "",
-      "box_download_checksum_type": ""
-    }
-  ]
-}
+## `controlrepo/.rizzo.yaml`
+
+```yaml
+---
+defaults:
+  bootstrap_script_path: "bootstrap_puppet4.sh"
+  bootstrap_script_args: "-l -f `hostname -f`"
+  bootstrap_guest_path: "/tmp/bootstrap"
+  boot_timeout: 500
+  box: "centos7.box"
+  box_url: "https://vagrantboxes/centos7.box"
+  box_download_checksum: "3764a2c4ae3829aa4b50971e216c3a03736aafb2"
+  box_download_checksum_type: "sha1"
+  memory: "1024"
+  netmask: "255.255.255.0"
+  update_packages: true
+  update_packages_command: "yum -y update"
+  shutdown: true
+  shutdown_command: "/sbin/shutdown -h now"
+nodes:
+  - name: "infra-puppetca"
+    hostname: "infra-puppetca1.example.org"
+    forwarded_ports:
+    - guest: "8140"
+      host: "8140"
+    ip: "172.16.100.5"
+    memory: "2048"
+  - name: "infra-puppet"
+    hostname: "infra-puppet1.example.org"
+    ip: "172.16.100.6"
+    memory: "2048"
+  - name: "www-api"
+    hostname: "www-api1.example.org"
+    ip: "172.16.100.7"
+  - name: "mssql1"
+    hostname: "mysql1.example.org"
+    forwarded_ports:
+    - guest: "5985"
+      host: "5985"
+    ip: "172.16.100.8"
+    windows: true
+    gui: true
+    shutdown: false
+    update_packages: false
+    box: "mwrock/Windows2016"
+    box_url: ""
```

Comment:
instead of `""` would we use `null` or `~`

Reply:
Ah ya, I missed that in README.